Domain.Sql.Tests/SqlEventSourcedRepositoryTests.cs

```diff
@@ -98,14 +110,15 @@ protected override IStoredEvent CreateStoredEvent(string streamName, string type
                     StreamName = "Order",
                     Type = "UKNOWN",
                     Body = new { ShoeSize = 10.5 }.ToJson(),
-                    SequenceNumber = 1,
+                    SequenceNumber = 2,
                     AggregateId = orderId,
```

Comment:
This should be changed back, since it's very unusual to have a sequence start at 2.